lib/dependencies/WorkerDependency.js

```diff
@@ -25,10 +25,13 @@ class WorkerDependency extends ModuleDependency {
 	/**
 	 * @param {string} request request
 	 * @param {[number, number]} range range
+	 * @param {Object} workerDependencyOptions options
+	 * @param {string} workerDependencyOptions.publicPath public path for the worker
 	 */
-	constructor(request, range) {
+	constructor(request, range, workerDependencyOptions) {
 		super(request);
 		this.range = range;
+		this.options = workerDependencyOptions;
```

Comment:
This need to be added to hashing and serialization. Check other dependencies with extra properties for examples

Reply:
added